db/migrate/20171103010745_create_author_books.rb

```diff
@@ -0,0 +1,9 @@
+class CreateAuthorBooks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :author_books do |t|
```

Comment:
whole this migration could use `rename_table :authors_books, :author_books` instead